components/activity_and_insights/insights/insights.tsx

```diff
@@ -60,17 +59,12 @@ const Insights = () => {
 
     useEffect(() => {
         dispatch(selectChannel(''));
-        dispatch(suppressRHS);
```

Comment:
@kyeongsoosoo Was there a reason to remove this and the unsuppress from insights?

Reply:
@BenCookie95  It seems that unnecessary changes have occurred in the process of returning the commit. Would it be better to open a related PR?

Reply:
Yeah it would be great if we could open a related PR

Reply:
I think I've commented under all  the insights related stuff but is there other stuff that shouldn't have been removed/added?

Reply:
@hmhealey What's the process for this? Would it be easier to revert this merge because there are a number of things that got accidentally removed

https://github.com/mattermost/mattermost-webapp/pull/11138
https://github.com/mattermost/mattermost-webapp/pull/11144

Reply:
I think it will be resolved if this commit(72319d65921d9971dc2a13d852eed3a75451dccf) is reverted.
Sorry for the mistake..!

Reply:
Yeah, reverting the whole thing and resubmitting it seems like it would be the easiest. Hopefully it's not too difficult to recreate the intended changes then.

@kyeongsoosoo Let us know if you need any help with that

Reply:
Should I open a PR with the same ticket?